packages/flutter/lib/src/material/chip.dart

```diff
@@ -1428,6 +1432,37 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
         ),
       ),
     );
+    switch(theme.materialTapTargetSize) {
+      case MaterialTapTargetSize.expanded:
+        result = new ConstrainedBox(
+          constraints: const BoxConstraints(minWidth: 48.0, minHeight: 48.0),
+          child: new Center(
+            child: result,
+            widthFactor: 1.0,
+            heightFactor: 1.0,
+          ),
+        );
+
```

Comment:
let's remove this blank line, otherwise it looks like the `canTap` branch is more closely related to the `case MaterialTapTargetSize.collapsed` line than the `case MaterialTapTargetSize.expanded` line.

Reply:
Fixed